### ISSUE TITLE
Segmentation Fault on HTTP 400 return code when clicking on video

### ISSUE THREAD

**Issue by Jkillelea** [vinszent/gnome-twitch#370]
```
[22:38:05] Warning - GNOME-Twitch : {GtTwitch:336} Received unsuccessful response from url 'http://api.twitch.tv/api/channels/dota2ti/access_token' with code '400' and body '{"error":"Bad Request","status":400,"message":"Requests must be made over SSL"}'
[22:38:05] Warning - GNOME-Twitch : {GtTwitch:594} Error getting stream access token for channel 'dota2ti' because: Received unsuccessful response from url 'http://api.twitch.tv/api/channels/dota2ti/access_token' with code '400' and body '{"error":"Bad Request","status":400,"message":"Requests must be made over SSL"}'
Segmentation fault (core dumped)
```

Looks like Twitch has changed their backend requirements?

**Comment by zabi94** [comment]
```
[18:42:09] Warning - GNOME-Twitch : {GtTwitch:336} Received unsuccessful response from url 'http://api.twitch.tv/api/channels/hypnotizd/access_token' with code '400' and body '{"error":"Bad Request","status":400,"message":"Requests must be made over SSL"}'
[18:42:09] Warning - GNOME-Twitch : {GtTwitch:594} Error getting stream access token for channel 'hypnotizd' because: Received unsuccessful response from url 'http://api.twitch.tv/api/channels/hypnotizd/access_token' with code '400' and body '{"error":"Bad Request","status":400,"message":"Requests must be made over SSL"}'
Errore di segmentazione (core dump creato)
```
Same error on my pc as well
Ubuntu 18.04/gnome-twitch 0.4.1-2/GStreamer-OpenGL

**Comment by monte-monte** [comment]
Just installed. Same error here. Ubuntu 18.04

**Comment by vinszent** [comment]
This was solved in 0.4.2, please ask the maintainers of that package to update it.
